nursery/encrypt-data-using-openssl-dsa.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: encrypt data using openssl DSA
+    namespace: data-manipulation/encryption/dsa
+    authors:
+      - "Ana06"
+    scope: function
+    references:
+      - https://github.com/openssl/openssl/blob/fdc5043d58900663b493147298e64f11353b35fe/crypto/objects/obj_dat.h
+  features:
+    - and:
+      - os: linux
```

Comment:
i think this code can be statically linked into programs on Windows/MacOS/etc. too, right? so can we remove this check?